cypress/e2e/ResultsTSV.cy.ts

```diff
@@ -37,10 +37,51 @@ describe('Results TSV', () => {
       const rows = fileContent.split('\n');
 
       const datasetProtected = rows[1];
-      const datasetNotProtected = rows[2];
+      const datasetNotProtected = rows[3];
 
       expect(datasetProtected.split('\t')[7]).to.equal('protected');
-      expect(datasetNotProtected.split('\t')[7]).to.equal('/ds004116/sub-300100');
+      expect(datasetNotProtected.split('\t')[8]).to.equal('/ds004116/sub-300101');
+    });
+  });
+  it('Checks whether the rows in the participant.tsv file generated according to session_type', () => {
+    cy.intercept('query/?*', unprotectedResponse).as('call');
+    cy.visit('/');
+    cy.get('[data-cy="submit-query-button"]').click();
+    cy.wait('@call');
+    cy.get('[data-cy="select-all-checkbox"]').find('input').check();
+    cy.get('[data-cy="dataset-level-download-results-button"]').click();
+    cy.get('[data-cy="participant-level-download-results-button"]').click();
+    cy.readFile('cypress/downloads/participant-level-results.tsv').then((fileContent) => {
```

Comment:
just want to mention how nice it is that with the very expressive `data-cy` selectors, the test is almost as readable as plaintext here. Very cool! I get a pretty clear idea what's going on